src/vm/docs/mod.rs

```diff
@@ -431,8 +431,8 @@ const FETCH_ENTRY_API: SpecialAPI = SpecialAPI {
 The value is looked up using `key-tuple`.
 If there is no value associated with that key in the data map, the function returns a `none` option. Otherwise,
 it returns `(some value)`.",
-    example: "(expects! (map-get names-map (tuple (name \"blockstack\"))) (err 1)) ;; Returns (tuple (id 1337))
-(expects! (map-get names-map ((name \"blockstack\"))) (err 1)) ;; Same command, using a shorthand for constructing the tuple
+    example: "(unwrap (map-get names-map (tuple (name \"blockstack\")))) ;; Returns (tuple (id 1337))
```

Comment:
Typo -- should be `unwrap!`?

Reply:
No -- `unwrap!` is the short-return variant, and would have an additional argument to supply the failure case short-return. `unwrap` without the exclamation point is a panicking variant. I'm not fully convinced that this is the correct naming scheme.

Reply:
I think what we can do is rename `unwrap` and `unwrap-err` to `unwrap-panic` and `unwrap-err-panic` to make the distinction more clear between the panic variant and the short return variant (`unwrap!` and `unwrap-err!`).

Reply:
So in this example, this should be `unwrap-panic`?